tools/somalier_ancestry.cwl

```diff
@@ -0,0 +1,51 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: somalier-ancestry
+doc: "Tool to calculate ancestry using sites and somalier inputs"
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 32000
+    coresMin: 16
+  - class: DockerRequirement
+    dockerPull: 'brentp/somalier:v0.2.15'
+baseCommand: [tar, -xzf]
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: >-
+      && somalier ancestry
+      $(inputs.untar_sites_prefix)/*.somalier
+  - position: 2
+    shellQuote: false
+    valueFrom: >-
+      ++ $(inputs.input_somalier[0].dirname)/*.somalier
```

Comment:
This feels dubious to me. I'm worried that this file list will not always have files that are in the same directory. Does the program require they all be in the same directory?

Reply:
Yes, it seems they should all be in the same dir: https://github.com/brentp/somalier/wiki/ancestry